src/functions/includegraphics.js

```diff
@@ -85,6 +85,10 @@ defineFunction({
             alt = alt.substring(0, alt.lastIndexOf('.'));
         }
 
+        if (!parser.settings.isTrusted("\\includegraphics", src)) {
```

Comment:
I think we should provide context information as the second argument like:
```
{
    type: "url",
    url: "https://katex.org/image.png",
    protocol: "https",
    ...
}
```
or
```
{
    type: "html",
    class: "my-class",
}
```
to allow setting across commands and prevent mis-parsing the url.